tests/bud.bats

```diff
@@ -2432,3 +2432,28 @@ EOF
   run_buildah 125 bud --network=bogus --signature-policy ${TESTSDIR}/policy.json -t ${target} ${TESTSDIR}/bud/containerfile
   expect_output "error checking for network namespace: stat bogus: no such file or directory"
 }
+
+@test "bud test --stdin" {
+  _prefetch alpine
+  mytmpdir=${TESTDIR}/my-dir
+  mkdir -p ${mytmpdir}
+cat > $mytmpdir/Containerfile << _EOF
+FROM alpine
+RUN read -t 1 x; echo test got \<\$x\>
+_EOF
+
+  run_buildah bud --stdin -t testbud --signature-policy ${TESTSDIR}/policy.json ${mytmpdir} <<< input
+  expect_output --substring "test got <input>"
+}
+
+@test "bud test no --stdin" {
+  _prefetch alpine
+  mytmpdir=${TESTDIR}/my-dir
+  mkdir -p ${mytmpdir}
+cat > $mytmpdir/Containerfile << _EOF
+FROM alpine
+RUN read -t 1 x
+_EOF
+
+  run_buildah 1 bud -t testbud --signature-policy ${TESTSDIR}/policy.json ${mytmpdir} <<< input
```

Comment:
There's a common pattern in buildah tests that I find odd: just test one little thing at a time. Given the substantial setup costs for each test, doesn't it make sense to occasionally bundle tests together? Like, you could combine with with --stdin: it's the same Containerfile, same setup, just a one-word difference in the command. You could first run the error one (perhaps adding `expect_output --substring 'error building at STEP "RUN read -t 1 x":'`), then run the one with `--stdin`. 

I realize there are pedantic voices who will say "write microtests so that every molecular component can be individually checked!!1!". Those voices are often silent when we need to deal with CI timeout errors or simply the development hassle of waiting hours and hours for CI runs. I am rabidly in favor of multipurpose tests: they are, most of the time, Good Enough.

Reply:
No problem.  I am just happy that I added some tests for it.  I will consolidate.